lib/get-test-dir.js

```diff
@@ -0,0 +1,19 @@
+// LICENSE : MIT
+"use strict";
+var normalizeDir = require("./normalize-dir");
+/**
+ * Return `dir/` string
+ * always suffix with `/`
+ * @param {object} pkg
+ * @returns {string}
+ */
+function getTestDirFromPkg(pkg) {
+    if (pkg &&
+        typeof pkg.directories === 'object' &&
+        typeof pkg.directories.test === 'string') {
+        return normalizeDir(pkg.directories.test);
+    }
+    return "test/"; // default value
```

Comment:
I think default value should not define in this function. 

* before

```javascript
function getTestDirFromPkg(pkg) {
    if (pkg &&
        typeof pkg.directories === 'object' &&
        typeof pkg.directories.test === 'string') {
        return normalizeDir(pkg.directories.test);
    }
    return "test/";
}
```

* after

```javascript
var defaultTestDir = "test/";
function getTestDirFromPkg(pkg) {
    if (pkg &&
        typeof pkg.directories === 'object' &&
        typeof pkg.directories.test === 'string') {
        return normalizeDir(pkg.directories.test);
    }
    return defaultTestDir;
}
```